hardware/sim/iruntrace

```diff
@@ -0,0 +1 @@
+simvision ies/latest/capiWave.shm
```

Comment:
Do you by purpose call the xsim waveform viewer with preloaded signals while you do not do that for irun/xcelium?

Reply:
```
simvision -wave -input <path to .swvf> ies/latest/capiWave.shm
```

Reply:
I had tried wave and input but didn't thought about the combination! Thx much. 
Tested it ok. This is indeed much better. Thx